core/common/build.gradle.kts

```diff
@@ -22,4 +22,7 @@ plugins {
 dependencies {
     implementation(libs.kotlinx.coroutines.android)
     testImplementation(project(":core:testing"))
-}
\ No newline at end of file
+}
+android {
```

Comment:
Nit: should we place the `android` block above `dependencies` to stay consistent?

Reply:
yeah probably, I used an automatic tool to make the changes but I'll go and change that

Reply:
Done.